### PR TITLE
Updates h2,h3 and body styling.

### PR DESCRIPTION
**Pull request by kchens**

### CODE REVIEW

**Comment by TheAlphaNerd** [comment]
can you please rebase this against develop and force push :D

File: app/content/what-is-new.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ##What's new?
 
 
-###Mixed Mode: 
+###MIXED MODE:
```
Comment (TheAlphaNerd) [inline app/content/what-is-new.md]: This should be done with CSS not manually

File: app/content/what-is-new.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
 
 Famous now unites DOM and WebGL under a single coordinate system. Borrowing concepts from modern video game engines, Famous structures applications in a scene graph hierarchy that makes manipulating HTML and WebGL together simple.
 
 
-###Rebuilt Architecture: 
+###REBUILT ARCHITECTURE:
```
Comment (TheAlphaNerd) [inline app/content/what-is-new.md]: see above

**Comment by TheAlphaNerd** [comment]
Rick made a few changes which overlap with this. You should rebase against develop and make sure that any dupes are removed.

**Comment by kchens** [comment]
I'm guessing I should rebase, and delete the two commits referring to peak..?

**Comment by rickarm** [comment]
@kchens sorry for the conflicts

**Comment by TheAlphaNerd** [comment]
landed as 

f0d24521fb3c746eac674c5e251675fef679b15d
0785be940da5f6d3a90d69bfe47a9d4ec8cfe419
85cd046d44bca4ef2d7db84d4abec515ba95009d

**Pull request closed without merging**